src/types.rs

```diff
@@ -420,6 +450,20 @@ pub struct BeaconProposerRegistration {
     pub fee_recipient: ExecutionAddress,
 }
 
+#[derive(Debug, Serialize, Deserialize)]
+pub struct ExpectedWithdrawals {
+    pub execution_optimistic: bool,
+    pub finalized: bool,
+    pub data: Vec<Withdrawal>,
+}
```

Comment:
I usually have been decoding this as `Value<Vec<Withdrawal>>` just so we don't have to worry about the exact metadata that could come along with it

so you can drop this type and just update the caller to use what I wrote here